vgen/src/main/resources/io/gapi/vgen/java/java_gapic.yaml

```diff
@@ -1,8 +1,10 @@
 type: io.gapi.vgen.ConfigProto
 language: java
-language_provider: io.gapi.vgen.java.JavaGapicLanguageProvider
-snippet_files:
-- main.snip
-- settings.snip
-fragment_files:
-- fragment.snip
+templates:
+- language_provider: io.gapi.vgen.java.JavaGapicCodeLanguageProvider
+  snippet_files:
+  - main.snip
+  - settings.snip
+- language_provider: io.gapi.vgen.java.JavaGapicFragmentLanguageProvider
+  snippet_files:
+  - fragment.snip
```

Comment:
When this change is pushed, artman _will_ generate fragments for Java when you run GAPIC. However, I think it is better to break up the configuration files than to change toolkit.